x/inflation/v1/keeper/inflation_test.go

```diff
@@ -162,24 +184,25 @@ func (suite *KeeperTestSuite) TestBondedRatio() {
 			"not mainnet",
 			false,
 			func() {},
-			math.LegacyMustNewDecFromStr("0.999900009999000099"),
+			math.LegacyMustNewDecFromStr("0.000029999100026999"),
```

Comment:
Why does this value change from the previous test?

Reply:
this type of values change because the setup changed